control/lti.py

```diff
@@ -405,8 +459,8 @@ def evalfr(sys, x):
 
     Notes
     -----
-    This function is a wrapper for StateSpace.evalfr and
-    TransferFunction.evalfr.
+    This function is a wrapper for StateSpace.__call__ and
+    TransferFunction.__call__.
```

Comment:
Tell Sphinx to link these

```suggestion
    This function is a wrapper for :meth:`StateSpace.__call__` and
    meth:`TransferFunction.__call__`.
```